projects/packages/forms/src/blocks/shared/hooks/use-form-wrapper.js

```diff
@@ -6,29 +6,30 @@ import { __ } from '@wordpress/i18n';
 import { FORM_BLOCK_NAME } from '../util/constants';
 
 export default function useFormWrapper( { attributes, clientId, name } ) {
-	const BUTTON_BLOCK_NAME = 'jetpack/button';
-	const SUBMIT_BUTTON_ATTR = {
-		text: __( 'Submit', 'jetpack-forms' ),
-		element: 'button',
-		lock: { remove: true },
-	};
-
 	const { replaceBlock, __unstableMarkNextChangeAsNotPersistent } = useDispatch( blockEditorStore );
+	const { getBlocks } = useSelect( blockEditorStore );
 
-	const parents = useSelect( select => {
-		return select( blockEditorStore ).getBlockParentsByBlockName( clientId, FORM_BLOCK_NAME );
-	} );
+	const parentForms = useSelect(
+		select => {
+			return select( blockEditorStore ).getBlockParentsByBlockName( clientId, FORM_BLOCK_NAME );
+		},
+		[ clientId ]
+	);
```

Comment:
Some small unrelated changes here - the `useSelect` was missing its deps, which I think makes it run the callback on every render 😬.

Also renamed `parents` to `parentForms` for a little more clarity.